tests/test_model_serializer.py

```diff
@@ -933,8 +933,8 @@ class TestSerializer(serializers.ModelSerializer):
 
             class Meta:
                 model = OneFieldModel
-                read_only_fields = ('char_field', 'non_model_field')
-                fields = read_only_fields
+                read_only_fields = ('char_field',)
+                fields = read_only_fields + ('non_model_field', )
```

Comment:
Shouldn't this make a test or two fail because `non_model_field` isn't actually specified as read-only anymore?